src/main/java/Sayo.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
```

Comment:
maybe you can set some of the functionalities like greetings and getting inputs as their own methods to improve readability